packages/plugins/on-resolve/src/index.ts

```diff
@@ -61,6 +68,7 @@ export function useOnResolve<PluginContext extends Record<string, any> = {}>(
         if ((!opts.skipIntrospection || !isIntrospectionType(type)) && isObjectType(type)) {
           for (const field of Object.values(type.getFields())) {
             if ((field as { [hasWrappedResolveSymbol]?: true })[hasWrappedResolveSymbol]) continue;
+            if (opts.skipDefaultResolvers && !field.resolve) continue;
```

Comment:
Perhaps we can also check that if it exists, it is different from `defaultFieldResolver` from `graphql` package ?